README.md

```diff
@@ -22,6 +22,23 @@ As opposed to a classic database server such as PostgreSQL or MySQL where querie
 
 `concrete-datastore` can be seen as a NoSQL server or as a Firebase alternative.
 
+## Quick start
+
+![Term sheet sample](https://concrete-datastore.netsach.org/en/latest/assets/mini-term-sample.svg)
```

Comment:
Did you check that the headers will be correctly set ? 

e.g. 

raw data without correct header :
https://raw.githubusercontent.com/paschembri/concrete-datastore/add-simple-quickstart/docs/assets/mini-term-sample.svg

data sanitized with stil incorrect headers (animation does not work)
https://raw.githubusercontent.com/paschembri/concrete-datastore/add-simple-quickstart/docs/assets/mini-term-sample.svg?sanitize=true